pcsx2/CDVD/Windows/DriveUtility.cpp

```diff
@@ -0,0 +1,64 @@
+//Copyright (C) 2020 PCSX2 Dev Team
+//Copyright (c) David Quintana <DavidQuintana@canal21.com>
+//
+//This library is free software; you can redistribute it and/or
+//modify it under the terms of the GNU Lesser General Public
+//License as published by the Free Software Foundation; either
+//version 3.0 of the License, or (at your option) any later version.
+//
+//This library is distributed in the hope that it will be useful,
+//but WITHOUT ANY WARRANTY; without even the implied warranty of
+//MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+//Lesser General Public License for more details.
+//
+//You should have received a copy of the GNU Lesser General Public
+//License along with this library; if not, write to the Free Software
+//Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA
+//
```

Comment:
Imo the formatting of this copyright header can be better so it matches the other ones.